spec/yaml/swiftnav/sbp/gnss_signal.yaml

```diff
@@ -16,6 +16,18 @@ include:
   - types.yaml
 definitions:
 
+ - GnssSignal16:
```

Comment:
Are we going to keep `GnssSignal` around? Are we not going to deprecated it? If we do want to keep both around (which sounds confusing - but I understand as there are lots of message that depend on it current), maybe a different name here other than a reference to its size - `ObsGnssSignal`? And if that's the case, maybe leave it with the obs message.